setup.py

```diff
@@ -42,21 +42,19 @@ def get_version(filename):
     packages=find_packages('src'),
     zip_safe=False,
     install_requires=[
-        'pyyaml<=5.2;python_version=="3.4"',
-        'pyyaml;python_version!="3.4"',
+        'pyyaml',
         'six~=1.11',
         'aws-sam-translator>=1.25.0',
-        'jsonpatch;python_version!="3.4"',
-        'jsonpatch<=1.24;python_version=="3.4"',
+        'jsonpatch',
         'jsonschema~=3.0',
-        'pathlib2>=2.3.0;python_version<="3.4"',
-        'importlib_resources~=1.0.2;python_version=="3.4"',
-        'importlib_resources~=1.4;python_version<"3.7" and python_version!="3.4"',
+        'pathlib2>=2.3.0;python_version<"3.5"',
+        'importlib_resources~=1.4;python_version<"3.7"',
         'networkx~=2.4;python_version>="3.5"',
         'networkx<=2.2;python_version<"3.5"',
+        'pyrsistent<=0.16.0;python_version<"3.5"',
```

Comment:
for [Python 2](https://stackoverflow.com/questions/63802032/cfn-lint-failing-while-validating-cloudformation-templates)